specification/search/data-plane/Azure.Search/preview/2024-03-01-Preview/searchservice.json

```diff
@@ -3145,7 +3145,10 @@
         "Edm.DateTimeOffset",
         "Edm.GeographyPoint",
         "Edm.ComplexType",
-        "Edm.Single"
+        "Edm.Single",
+        "Edm.Float16",
```

Comment:
Feedback from API review: consider naming it Edm.Half

Reply:
Updated to use Edm.Half